lessons/Early-Math/identity-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,108 @@
+= The Identity Property
+
+@description{This lesson introduces students to the Identity Property by analyzing and exploring Circles of Evaluation.}
+
+@lesson-prereqs{equivalence}
+
+@keywords{Circle of Evaluation, Identity Property}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Recognize that multiplying and dividing by one does not affect the value of an expression.
+* Recognize that adding or subtracting zero does not affect the value of an expression.
+* Demonstrate their understanding of the Identity Property with and without variables via Circles of Evaluation, numeric expressions, and words.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's explore the Identity Property and Circles of Evaluation!
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+|===
+
+== The Identity Property
+
+=== Overview
+
+Students analyze pairs of Circles to see if they represent equivalent expressions. The Circles make visible the structural changes that occur when we apply the Identity Property.
+
+=== Launch
+
+Build on prior knowledge by reminding students of the activity that they completed in the previous lesson. You might even refer to the list of category names that the class created, and then draw students’ attention to any category names that allude to or suggest the commutativity. (Any mention of “changing the order” is a strong hint that students’ have clued into the Commutative Property!)
+
+[.lesson-instruction]
+- Now, turn to @printable-exercise{discover-identity.adoc}. Look at the first problem. Translate the expressions in words into Circles of Evaluation. What do you Notice? What do you Wonder?
+** _Possible responses: All three Circles include @math{12 + 4}. All three Circles evaluate to @math{16}._
+- Complete the worksheet by filling in the blanks for the remaining Circles. Avoid putting the same operator more than once.
+- Keisha says that when she multiplies a number by zero, she always gets that number. Do you agree?
+** _No, multiplying by zero produces zero._
+- What did you observe about the Identity Property?
+** _Possible responses: I can multiply or divide by @math{1} and get an equivalent result. I can add or subtract @math{0} and get an equivalent result._
```

Comment:
You're mentioning division and subtraction here. Feels like that comes out of nowhere...